### PR TITLE
Adds a new emergency shuttle: The Crowd-Source Shuttle 

### PR DESCRIPTION
**Pull request by maxion12345**
# Document the changes in your pull request

Adds a new emergency shuttle.

This shuttle was the one built during round 48641:
9k2J6ac
ZzBsiYX

Some items removed to preserve gameplay (The artillery, the BS launchpad, Cargo lathe)
This shuttle features a rage cage (with a generator to electrify it), a medical area, and a fully functional cargo bay. 
# Spriting
n/a
# Wiki Documentation
n/a
# Changelog

:cl:  
rscadd: Added the Crowd-Sourced Shuttle. This shuttle was built from the BYOS on round 48641
/:cl:

### CODE REVIEW

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -378,6 +378,15 @@
...
 	credit_cost = 7000
 
+/datum/map_template/shuttle/emergency/foureightsixfourone
+	suffix = "48641"
+	name = "Crowd-Sourced Emergency Shuttle"
+	description = "The Crowd-Sourced Emergency Shuttle is the product of a Centcomm initiative to have crews design their own emergency shuttles. \
+		Due to its strange construction, this shuttle offers some amenities not available on other shuttles. \
+		The most notable additions are: An electrified arena, A fully functional cargo bay, and a club."
+	admin_notes = "The emergency shuttled created during Round 48641. "
+	credit_cost = 1
```
Comment (ynot01) [inline code/datums/shuttles.dm]: image

Comment (maxion12345) [reply code/datums/shuttles.dm]: its late ok

**Comment by athena413** [comment]
I wasnt there for it but the look of this shuttle makes me happy in a way that I cant describe

Maybe take out the spear or circular saw though, and if that console is an express supply console consider swapping it out for something else. Im fine with it but im not a maintainer or a reviewer of any kind

**Comment by Hislyna** [comment]
Should make an emagged ver of it with all the cool original shiz on it too

**Comment by maxion12345** [comment]
Just realized there's no brig in this. Might need to look into further modification. If anyone has suggestions, please let me know. 

Briefly considering having the brig tiles be in the rage cage, so security can throw the prisoners in there to fight or something. It'd be a pretty radical take on securing prisoners for sure.

**Comment by MajManatee** [comment]
Not all shuttles need a brig. See: Roller dome

**Comment by N3D6** [comment]
This is epic

**Comment by maxion12345** [comment]
> Not all shuttles need a brig. See: Roller dome

Good point.

**Comment by maxion12345** [comment]
Wait did linter actually fail because I put an extra m in centcom? lmaooo

**Comment by JohnFulpWillard** [comment]
Why are we adding BYOS shuttles as purchaseable shuttles doesn't that take away from the building aspect of it

**Comment by maxion12345** [comment]
> Why are we adding BYOS shuttles as purchaseable shuttles doesn't that take away from the building aspect of it

Huh? No. You are free to construct other shuttles. 

 The amount of effort put into the construction of a decent shuttle is difficult to pull off, especially if your construction efforts are hampered by antagonists or an early shuttle call. It's rare for something like this to ever actually be constructed, and it has its own unique organic construction that comes from building in-game with resources collected by you instead of being mapped out with perfect geometry on the map editor.

So, no. Adding this does not take away from the building aspect, no more then being able to buy any shuttle does. If anything, I'd hope to encourage people to attempt to replicate these rare moments where a crew pulls together to construct something unique.

**Review by ynot01 — APPROVED** [review]

**Review by Ethan4303 — CHANGES_REQUESTED** [review]
everythings good, but i'd like to see the buttons, shutters, and the blastdoor to have their ID changed instead of BYOS(shutter)/BYOS(blastdoor), but i like it

**Review by Ethan4303 — APPROVED** [review]
looks good

**Pull request merged**